AirshipKit/AirshipKit/ios/UAInAppMessageDismissButton.m

```diff
@@ -29,6 +29,8 @@ - (instancetype)initWithIconImageName:(NSString *)iconImageName color:(UIColor *
     if (self) {
         self.userInteractionEnabled = YES;
 
+        self.accessibilityLabel = @"Dismiss";
```

Comment:
Should this be a localized string?

Reply:
Yes, but I wasn't quite sure what the right way would be since I can't provide the translations.

Reply:
Just checking. Does not look like we have a string for that currently so we will have to get it localized. I would assume english is better than nothing, so I am +1 on merging/releasing this without the strings.